horizon_hwm_store/horizon_hwm_store.py

```diff
@@ -213,6 +214,24 @@ def check(self) -> HorizonHWMStore:
         self._get_namespace_id()
         return self
 
+    def create_namespace(self, name: str) -> None:
+        """
+        Create a namespace.
+
+        Parameters
+        ----------
+        name : str
+            The name of the namespace to create.
+
+        Examples
+        --------
+        .. code:: python
+
+            hwm_store.create_namespace(namespace_unique_name)
+        """
+        if not self.client.paginate_namespaces(query=NamespacePaginateQueryV1(name=name)).items:
+            self.client.create_namespace(NamespaceCreateRequestV1(name=name))
```

Comment:
Please set here `self._namespace_id` to avoid fetching namespace by name second time. Maybe even more repeating `self.client.paginate_namespaces(query=NamespacePaginateQueryV1(name=name)).items` to its own method

Reply:
`def _get_namespace_id` can reuse this method